python/inline/conftest.py

```diff
@@ -203,6 +206,11 @@ class MalformedException(Exception):
     pass
 
 # TODO: define a timeout excpetion
```

Comment:
Can you help remove all the TODO about timeout?

Reply:
resolved